oci/private/pull.bzl

```
@@ -333,6 +345,7 @@ def _oci_alias_impl(rctx):
     available_platforms = []
 
     manifest, _, digest = downloader.download_manifest(rctx.attr.identifier, "mf.json")
+    rctx.file("digest.txt", digest + "\n")
```

what's the reason for `\n` here?  I would expect `$(cat <path_to_digest_file>)` to have no trailing NLs? can we remove this?

---

Yeah we can. I though having an EOL character at the end of the file may make it nice.

```suggestion
    rctx.file("digest.txt", digest)
```